lib/enigma.rb

```diff
@@ -0,0 +1,8 @@
+class Enigma
+  attr_reader :shift_generator, :character_set
+
+  def initialize
+    @character_set = ("a".."z").to_a << " "
+    @shift_generator = ShiftGenerator.new
+  end
+end
```

Comment:
this has yet to be refactored, enigma will be tasked to take input messages and output messages